config/packages/package-operator/components/hosted-cluster/manifest.yaml.tpl

```
@@ -27,6 +27,9 @@ spec:
   config:
     openAPIV3Schema:
       properties:
+        logLevel:
+          description: Log levels for package operator
```

It think it might be better to use the same description as in the CLI flags for consistency. But feel free to disagree. :slightly_smiling_face: 
```suggestion
          description: Log level. Default is -1 (warn). Higher numbers increase verbosity (e.g., 0 = info, 1 = debug)
```